Source/DataSources/CorridorGeometryUpdater.js

```diff
@@ -176,8 +176,10 @@ define([
     CorridorGeometryUpdater.prototype._getIsClosed = function(options) {
         var height = options.height;
         var extrudedHeight = options.extrudedHeight;
-        var isExtruded = defined(extrudedHeight);
-        return (!isExtruded && height === 0) || (isExtruded && extrudedHeight !== height);
+        if (defined(extrudedHeight)) {
+            return extrudedHeight !== height || height === 0;
+        }
+        return height === 0;
```

Comment:
I find this `if` hard to read, because we repeat the `height === 0` in two places, even though they are the same check.  Can we just turn this into a one liner?

`return height === 0 || (defined(extrudedHeight) && extrudedHeight !== height)`

Reply:
Thanks, I was trying to figure out how to do this but boolean logic was failing me haha